test/Microsoft.AspNet.Mvc.TagHelpers.Test/LinkTagHelperTest.cs

```diff
@@ -202,21 +202,21 @@ public void PreservesOrderOfSourceAttributesWhenRun()
         {
             // Arrange
             var context = MakeTagHelperContext(
-                attributes: new Dictionary<string, object>
+                attributes: new TagHelperAttributes
                 {
-                    ["rel"] = new HtmlString("stylesheet"),
-                    ["data-extra"] = new HtmlString("something"),
-                    ["href"] = "test.css",
-                    ["asp-fallback-href"] = "test.css",
-                    ["asp-fallback-test-class"] = "hidden",
-                    ["asp-fallback-test-property"] = "visibility",
-                    ["asp-fallback-test-value"] = "hidden"
+                    { "rel", new HtmlString("stylesheet") },
+                    { "data-extra", new HtmlString("something") },
+                    { "href", "test.css" },
+                    { "asp-fallback-href", "test.css" },
+                    { "asp-fallback-test-class", "hidden" },
+                    { "asp-fallback-test-property", "visibility" },
+                    { "asp-fallback-test-value", "hidden" },
```

Comment:
Invalid initializer member declarator when intermingling the two. Not going to look into why. Just going to leave for now, no harm.